contributors/devel/sig-testing/testing.md

```diff
@@ -25,6 +25,9 @@ passed both unit and integration tests.
 Kubernetes only merges pull requests when unit, integration, and e2e tests are
 passing, so it is often a good idea to make sure the e2e tests work as well.
 
+
+
+
```

Comment:
I'm guessing we don't need these extra lines?

Reply:
Correct! Fixed in https://github.com/kubernetes/community/pull/4170/commits/af108e4f6e1c819992843e4c8dde06ffb52d5e70